examples/acquiring_data/downloading_hmi.py

```diff
@@ -23,7 +23,6 @@
 
 result = Fido.search(a.Time('2014/11/20 00:00:00', '2014/11/20 00:04:00'),
                      a.jsoc.Series("hmi.M_720s"),
-                     a.jsoc.Keys(["T_REC, CROTA2"]),
```

Comment:
WIll these two columns be displayed in the results table now?

Reply:
I have added another section in this example which shows how to display specific keywords.